### PR TITLE
Readability improvements inbound pipeline

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
- [x] write shorter sentence

### CODE REVIEW

File: 10/umbraco-cms/implementation/default-routing/inbound-pipeline.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ versionTo: 10.0.0
 
 # Umbraco's request pipeline
 
-Umbraco's request pipeline is the process of building-up the URL for a node, resolving a request to a specified node, and ensuring that the right content is sent back.
+Umbraco's request pipeline building-up an URL, resolves the request, and returns the correct content.
```
Comment (sofietoft) [inline 10/umbraco-cms/implementation/default-routing/inbound-pipeline.md:8]: How about this instead:

```suggestion
Umbraco's request pipeline is the process of building up a URL, resolving the requests, and returning correct content.
```

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR @erikjanwestendorp !

I've added a suggestion to change the wording a bit! Let me know what you think 💪

File: 10/umbraco-cms/implementation/default-routing/inbound-pipeline.md (hunk 1 above)
Comment (erikjanwestendorp) [reply 10/umbraco-cms/implementation/default-routing/inbound-pipeline.md:8]: @sofietoft seems good to me 😄

**Comment by erikjanwestendorp** [comment]
@sofietoft Thanks for your review, just updated the PR 😄

**Comment by sofietoft** [comment]
Thanks @erikjanwestendorp ! 💪 
I'll merge the PR as soon as it passes the checks!

**Pull request merged**